build.gradle.kts

```diff
@@ -17,9 +17,6 @@ plugins {
 }
 
 allprojects {
-    group = rootProject.providers.gradleProperty("GROUP").get()
```

Comment:
after 3 days investigation I finally found that this 2 lines break build 😑

Reply:
I was guessing this too.